typescript/infra/scripts/funding/fund-relayers-from-deployer.ts

```diff
@@ -158,4 +178,49 @@ function getRelayerKeysFromSerializedAddressFile(path: string): AgentKey[] {
     .filter((key: AgentKey) => key.role === KEY_ROLE_ENUM.Relayer);
 }
 
+function submitFunderBalanceMetrics(balances: FunderBalance[]) {
+  const gatewayAddr = process.env['PROMETHEUS_PUSH_GATEWAY'];
+  if (!gatewayAddr) {
+    console.warn(
+      'Prometheus push gateway address was not defined; not publishing metrics.',
+    );
+    return;
+  }
+
+  const register = new Registry();
+  // TODO: get actual push gateway address
+  const gateway = new Pushgateway(gatewayAddr, [], register);
+  const gauge = new Gauge({
+    name: 'abacus_relayer_funder_balance',
+    help: 'Last known balance of native tokens for each chain of the relayer funder',
+    registers: [register],
+    labelNames: ['chain', 'address'],
```

Comment:
could we add an `'abacus_deployment'` label and set it to the `environment`, just like we have for other metrics?
Could you also add an `'abacus_context'` label and for now just hardcode it to `'abacus'`? Following https://github.com/abacus-network/abacus-monorepo/pull/691 (and sorta https://github.com/abacus-network/abacus-monorepo/pull/683) this is also present for our other infrastructure

Reply:
I assume this label will be set at a higher level, if not then absolutely.